spacy/tokens/span.pyi

```diff
@@ -119,6 +119,8 @@ class Span:
     kb_id: int
     ent_id: int
     ent_id_: str
+    id: int
+    id_: str
```

Comment:
Since `id` and `ìd_` are properties, it's cleaner to declare them like this:
```suggestion
    @property
    def id(self) -> int: ...
    @property
    def id_(self) -> str: ...
```
(We are a bit inconsistent here with the other properties, but will clean that up.)